.rubocop.yml

```diff
@@ -24,7 +24,7 @@ Style/DocumentationMethod:
     - lib/**/*.rb
 
 Style/StringLiterals:
-  EnforcedStyle: single_quotes
+  EnforcedStyle: double_quotes
```

Comment:
Can we wait until the gem is updated to entirely remove that section instead? Or we can ship and fix that later with the update.

Reply:
I'd prefer to ship now so we don't have to continue making changes / maintaining this PR as new strings get added. Since the current version doesn't specify a rule for `Style/StringLiterals`, we could just remove this now ahead of the new version being released, but it's nice to have it for autoformatting / external contributors IMHO. I'll go ahead and ship this, but leave a note that we should remove this rule when `rubocop-shopify` gets bumped.